### PR TITLE
Change "Skipping head block update" message to debug level   #4364 

### LINKED ISSUE #4364: Change "Skipping head block update" message to debug level
The message `Skipping head block update as justified checkpoint was updated while loading checkpoint state. Was 64667 (0x09991b31991f434eeb0e853263039977b6ed818b3ba9212ffedb996d8a179cee) but now 64668 (0x65769cb3ae74f772e40a86360c21ae7bb6d0ae6013fd9b86b0313a766417a6b5)` can be changed to be debug level.

To do this, will be just a matter of finding "Skipping head block update as justified" in code, and changing to use `.debug` function instead of `.info`

### PR DESCRIPTION
**Pull request by seroze**
## PR Description
fix https://github.com/ConsenSys/teku/issues/4364 
[4364]  Change log from info to warn 

## Documentation

- [ ] I thought about documentation and added the `documentation` label to this PR if updates are required.

## Changelog

- [ ] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

**Comment by seroze** [comment]
@rolfyone  do have look at this when you get time.

File: ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java [hunk 1]
```diff
@@ -119,7 +119,7 @@ private void initializeProtoArrayForkChoice() {
                       final Checkpoint justifiedCheckpoint =
                           recentChainData.getStore().getJustifiedCheckpoint();
                       if (!justifiedCheckpoint.equals(retrievedJustifiedCheckpoint)) {
-                        LOG.info(
+                        LOG.warn(
```
Comment (ajsutton) [inline ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java]: I think we want to go the other way on this and make it `LOG.debug`

**Review by ajsutton — COMMENTED** [review]
Sorry for the delay in reviewing this one - it's been the school holiday period in Australia and apparently we didn't keep up with PRs as well as we meant to.

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**